src/includes/platforms/configuration/integrations/redux.mdx

```diff
@@ -107,3 +107,13 @@ const sentryReduxEnhancer = Sentry.createReduxEnhancer({
   },
 });
 ```
+
+`attachReduxState` (Boolean)
+
+This is `true` by default. It attaches a file `redux_state.json` with Redux state to all error events sent to Sentry. It is beneficial where Redux states are too big. Note if the stateTransformer function is provided, it will attach the transformed state. If you don't want to attach the state to error events, set this to `false`.
```

Comment:
```suggestion
This is `true` by default. It attaches a file `redux_state.json` with Redux state to all error events sent to Sentry. It is beneficial where Redux states are too big. If the `stateTransformer` function is provided, it will attach the transformed state. If you don't want to attach the state to error events, set this to `false`.
```